bin-src/lib/getDependentStoryFiles.ts

```diff
@@ -172,8 +165,16 @@ export async function getDependentStoryFiles(
     bailReason: undefined,
   };
 
-  const changedPackageFiles = tracedFiles.filter(isPackageFile);
-  if (changedPackageFiles.length) ctx.turboSnap.bailReason = { changedPackageFiles };
+  const changedPackageFiles = tracedFiles.filter(isPackageLockFile);
+  if (changedPackageFiles.length) {
+    ctx.turboSnap.bailReason = { changedPackageFiles };
+    // If package.json dependencies changed, we still want to use the same TurboSnap bail reason
+    // for now.
+  } else if (ctx.git.packageManifestsWithDependencyChanges?.length) {
+    ctx.turboSnap.bailReason = {
+      changedPackageFiles: ctx.git.packageManifestsWithDependencyChanges,
```

Comment:
I didn't want to add a new bail reason to the data model since we'll likely soon be avoiding bailing for package.json changes altogether.